tribe-ext-pdf-tickets.php

```diff
@@ -223,18 +240,18 @@ public function init() {
 		 */
 		public function create_pdf_file_creation_deletion_triggers() {
 			// do_upload_pdf() when tickets are created
-			add_action( 'event_tickets_rsvp_attendee_created', [ $this, 'do_upload_pdf' ], 50, 1 );
+			add_action( 'event_tickets_rsvp_ticket_created', [ $this, 'do_upload_pdf' ], 50, 1 );
 
 			// Event Tickets: Tribe PayPal
 			add_action( 'event_tickets_tpp_attendee_created', [ $this, 'do_upload_pdf' ], 50, 1 );
 
+			// Event Tickets Plus: Easy Digital Downloads (EDD)
+			add_action( 'eddtickets-send-tickets-email', [ $this, 'do_upload_pdf' ], 50, 1 );
+
 			// Event Tickets Plus: WooCommerce
-			add_action( 'event_ticket_woo_attendee_created', [ $this, 'do_upload_pdf' ], 50, 1 );
```

Comment:
What was the reason for removing this? Just making sure it was intentional

Reply:
I added code comments to explain why different than other ticket providers